src/Contract.php

```diff
@@ -72,17 +78,22 @@ public function toPdf(HtmlConverterInterface $htmlConverter, array $data = [])
     /**
      * Generates file and sends a SignRequest
      * @param DocumentSignInterface $documentSign
+     * @return bool|void
      */
     public function sign(DocumentSignInterface $documentSign)
     {
-        if (!$this->file || !config('mvc-contracts.document_sign.enabled')) {
+        $config = config('mvc-contracts.document_sign');
+
+        if (!$this->file || !$config['enabled']) {
```

Comment:
You cannot say these config variables are always present. So always use Arr::get